packages/shared/src/lib/log.ts

```diff
@@ -149,9 +149,20 @@ export enum LogEvent {
   ScheduleStreakReminder = 'schedule streak reminder',
   // 404 page
   View404Page = '404 page',
-  // source
+  // Source Actions - start
+  BlockSource = 'block source',
+  UnblockSource = 'unblock source',
```

Comment:
Just verify with Ido if this is ok so it can be added to taxonomy sheet (also with other events related changes)